### PR TITLE
Perf baselines updates (m5d, m6a, vsock, network)

### PR DESCRIPTION
**Pull request by kalyazin**
## Changes

Update baselines for vsock and network for m5d and m6a.

## Reason

- Network latency improvement for m5d/4.14
- Network TCP throughput degradation for m5d/4.14
- Vsock throughput degradation for m6a/5.10

```
{
    "source": "HEAD~/tests/integration_tests/performance/configs/",
    "target": "HEAD/tests/integration_tests/performance/configs/",
    "test_network_latency_config_4.14.json": {
        "test": "network_latency",
        "kernel": "4.14",
        "cpus": [
            {
                "instance": "m5d.metal",
                "model": "Intel(R) Xeon(R) Platinum 8259CL CPU @ 2.50GHz",
                "stats": {
                    "pkt_loss": {
                        "target_diff_percentage": {
                            "mean": 0.0,
                            "stdev": 0.0
                        },
                        "delta_percentage_diff": {
                            "mean": 0.0,
                            "stdev": 0.0
                        }
                    },
                    "latency": {
                        "target_diff_percentage": {
                            "mean": -7.349206349206344,
                            "stdev": 0.29182184620398544
                        },
                        "delta_percentage_diff": {
                            "mean": 19.0,
                            "stdev": 2.8284271247461903
                        }
                    }
                }
            },
            {
                "instance": "m5d.metal",
                "model": "Intel(R) Xeon(R) Platinum 8175M CPU @ 2.50GHz",
                "stats": {
                    "pkt_loss": {
                        "target_diff_percentage": {
                            "mean": 0.0,
                            "stdev": 0.0
                        },
                        "delta_percentage_diff": {
                            "mean": 0.0,
                            "stdev": 0.0
                        }
                    },
                    "latency": {
                        "target_diff_percentage": {
                            "mean": -7.893048128342246,
                            "stdev": 1.1797717418727451
                        },
                        "delta_percentage_diff": {
                            "mean": 23.0,
                            "stdev": 14.142135623730951
                        }
                    }
                }
            }
        ]
    }
}{
    "source": "ac598701549e6649a8c043fe4083b5636735965e/tests/integration_tests/performance/configs/",
    "target": "tests/integration_tests/performance/configs/",
    "test_network_tcp_throughput_config_4.14.json": {
        "test": "network_tcp_throughput",
        "kernel": "4.14",
        "cpus": [
            {
                "instance": "m5d.metal",
                "model": "Intel(R) Xeon(R) Platinum 8259CL CPU @ 2.50GHz",
                "stats": {
                    "cpu_utilization_vcpus_total": {
                        "target_diff_percentage": {
                            "mean": 0.10012162124534578,
                            "stdev": 0.4331599719566124
                        },
                        "delta_percentage_diff": {
                            "mean": -0.016666666666666666,
                            "stdev": 1.2001647721150586
                        }
                    },
                    "throughput": {
                        "target_diff_percentage": {
                            "mean": -0.06272356343359392,
                            "stdev": 1.140603271281945
                        },
                        "delta_percentage_diff": {
                            "mean": 0.3333333333333333,
                            "stdev": 1.673995187270755
                        }
                    },
                    "cpu_utilization_vmm": {
                        "target_diff_percentage": {
                            "mean": 0.2960776532788734,
                            "stdev": 1.1838221921067151
                        },
                        "delta_percentage_diff": {
                            "mean": 0.18333333333333332,
                            "stdev": 2.166710125596311
                        }
                    }
                }
            }
        ]
    }
}{
    "source": "6eae211d11aa24bea9e001835fd77e911a937de8/tests/integration_tests/performance/configs/",
    "target": "tests/integration_tests/performance/configs/",
    "test_vsock_throughput_config_5.10.json": {
        "test": "vsock_throughput",
        "kernel": "5.10",
        "cpus": [
            {
                "instance": "m6a.metal",
                "model": "AMD EPYC 7R13 48-Core Processor",
                "stats": {
                    "throughput": {
                        "target_diff_percentage": {
                            "mean": -0.5681871089411127,
                            "stdev": 2.8783724052977147
                        },
                        "delta_percentage_diff": {
                            "mean": -0.03333333333333333,
                            "stdev": 1.9910720268468327
                        }
                    },
                    "cpu_utilization_vcpus_total": {
                        "target_diff_percentage": {
                            "mean": 0.2226067843070815,
                            "stdev": 1.7054311176703554
                        },
                        "delta_percentage_diff": {
                            "mean": -0.6333333333333333,
                            "stdev": 4.8243872816843965
                        }
                    },
                    "cpu_utilization_vmm": {
                        "target_diff_percentage": {
                            "mean": -0.5122319798964969,
                            "stdev": 1.6405121060255599
                        },
                        "delta_percentage_diff": {
                            "mean": -0.5666666666666667,
                            "stdev": 3.953901027255656
                        }
                    }
                }
            }
        ]
    }
}
```

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

- ~~[ ] If a specific issue led to this PR, this PR closes the issue.~~
- [x] The description of changes is clear and encompassing.
- ~~[ ] Any required documentation changes (code and docs) are included in this PR.~~
- ~~[ ] API changes follow the [Runbook for Firecracker API changes][2].~~
- ~~[ ] User-facing changes are mentioned in `CHANGELOG.md`.~~
- ~~[ ] All added/changed functionality is tested.~~
- ~~[ ] New `TODO`s link to an issue.~~
- [x] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

- [ ] This functionality can be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: tests/integration_tests/performance/configs/test_network_latency_config_4.14.json [hunk 1]
```diff
@@ -72,8 +72,8 @@
                                         "1vcpu_1024mb.json": {
                                             "Avg": {
                                                 "ping": {
-                                                    "delta_percentage": 17.1,
-                                                    "target": 0.225
+                                                    "delta_percentage": 38.1,
```
Comment (roypat) [inline tests/integration_tests/performance/configs/test_network_latency_config_4.14.json:75]: pretty big deltas in this file :/

**Review by roypat — APPROVED** [review]

**Comment by zulinx86** [comment]
Could you give the comparison result with https://github.com/firecracker-microvm/firecracker/blob/main/tools/compare_baselines/main.py ?
```
python3 tools/compare_baselines/main.py commit --tests vsock_throughput network_latency --codenames skylake cascadelake milan -s <source commit hash> -t <target commit hash>
```

**Comment by kalyazin** [comment]
> Could you give the comparison result with https://github.com/firecracker-microvm/firecracker/blob/main/tools/compare_baselines/main.py ?
> 
> ```
> python3 tools/compare_baselines/main.py commit --tests vsock_throughput network_latency --codenames skylake cascadelake milan -s <source commit hash> -t <target commit hash>
> ```

added

**Review by zulinx86 — APPROVED** [review]

File: tests/integration_tests/performance/configs/test_network_latency_config_4.14.json (hunk 1 above)
Comment (zulinx86) [reply tests/integration_tests/performance/configs/test_network_latency_config_4.14.json:75]: yeah, it seems we got some performance improvement, instead variation came with it.

**Pull request merged**